ratis-test/src/test/java/org/apache/ratis/grpc/TestGrpcOutputStream.java

```diff
@@ -31,9 +32,7 @@
 
 /**
  * Test {@link GrpcOutputStream}
- * TODO: {@link GrpcOutputStream} current has some bugs.
  */
-@Ignore
```

Comment:
Please run it many times to make sure there is no bug.

Reply:
@runzhiwang I wasn't clear enough before:

both `testKillLeader` and `testSimpleWrite` are not a case of transient failures. They failed deterministic. 

This PR fixed the root cause of `testSimpleWrite` and RATIS-1065 will track the fix for `testKillLeader`

I will fix both RATIS-1065 and RATIS-1064 to move them out of the category of transient failures to make it more clear.

Reply:
I also ran `TestGrpcOutputStream.testSimpleWrite` 30 times locally and without seeing a problem.